scripts/website/contentpreprocess.sh

```diff
@@ -1,37 +1,14 @@
 #!/usr/bin/env bash
 
-if ! [[ $0 == "scripts/websitepreprocess.sh" ]]; then
-  echo "must be run from repository root"
-  exit 255
+if ! [[ "$0" =~ "scripts/website/contentpreprocess.sh" ]]; then
```

Comment:
I think we discussed this a little bit ago and we were wondering, is this actually helping at all? Is it to ensure we output files in the correct location later?

Reply:
You're right. This isn't helping in any way. It's just that I thought it's a convention when writing bash scripts. Would effect changes right away. Thanks for bringing this up 🤗